### PR TITLE
Verbosify quickstart to incorporate more specific details

### PR DESCRIPTION
**Pull request by cherti**
This adds a little more information to the quickstart guide that help users that have never used the lua-capablilties of nginx to get things up and running.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,8 +21,17 @@ is also available via
 
 ## Quick start guide
 
-To track request latency broken down by server name and request count broken
-down by server name and status, add the following to `nginx.conf`:
+In case you use an external module for nginx-lua-support (e.g. the
+`libnginx-mod-http-lua` package on Debian) first add
```
Comment (knyar) [inline README.md]: As far as I can tell, this is not necessary. Debian package automatically adds a symlink to `/etc/nginx/modules-enabled` which enables the module (please see `/var/lib/dpkg/info/libnginx-mod-http-lua.postinst`).

File: README.md [hunk 2]
```diff
@@ -21,8 +21,17 @@ is also available via
...
+
+    load_module modules/ndk_http_module.so;
+    load_module modules/ngx_http_lua_module.so;
+
+to the beginning of `nginx.conf`.
+
+To then track request latency broken down by server name and request count
+broken down by server name and status, add the following to the `http` section
```
Comment (knyar) [inline README.md]: Specifying that you need to add this into the `http` section seems useful.

I'd suggest to adjust this PR to only change this paragraph, removing instructions related to enabling the module, since it will be done automatically if you are using the package manager.

**Review by knyar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cherti) [reply README.md]: I had to enable both modules on Debian Stretch to get it running. I just verified by commenting both lines and either one of them out and nginx fails to start unless both lines are present. I could rephrase it to a "in case you have problems with nginx not recognizing lua statements" or so, though, if you cannot reproduce this behavior?

Comment (cherti) [reply README.md]: This is with the `nginx-full` package, might be different with other nginx-packages? Which one are you running?

Comment (knyar) [reply README.md]: That's with `libnginx-mod-http-lua`. Have you looked at `/var/lib/dpkg/info/libnginx-mod-http-lua.postinst`?

Comment (cherti) [reply README.md]: I wasn't aware of that. I updated the description accordingly.

**Comment by knyar** [comment]
Thanks!
